From 75f007e96965e79d05887f3a21038bc876e6edee Mon Sep 17 00:00:00 2001
From: Michael Bestas <mikeioannina@gmail.com>
Date: Tue, 9 Apr 2013 20:22:21 +0300
Subject: [PATCH] DASH: increase as3676 light sensor max range

A test was run on many msm7x30 phones and the sensor reports maximum value of ~12276 lux
Increasing the max range we add the ability to the users to fully customize the light levels
The current max range limits us to tweaks up to 4900 lux, levels higher than this can't be customized by the user

Change-Id: I274777ae9517df346ab9fd1ff86ceaebbc5a98bf
Signed-off-by: Michael Bestas <mikeioannina@gmail.com>
---
 sensors/Sensors.mk |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sensors/Sensors.mk b/sensors/Sensors.mk
index c56153b..653231f 100644
--- a/sensors/Sensors.mk
+++ b/sensors/Sensors.mk
@@ -69,7 +69,7 @@ $(yes-var-compass-ak896xna)-files += ak896xna.c wrappers/ak896xna_sensors.c
 #
 
 $(SOMC_CFG_SENSORS_LIGHT_AS3676)-files += as3676_als.c
-$(SOMC_CFG_SENSORS_LIGHT_AS3676)-cflags += -DALS_CHIP_MAXRANGE=4900
+$(SOMC_CFG_SENSORS_LIGHT_AS3676)-cflags += -DALS_CHIP_MAXRANGE=13000
 
 ifneq ($(SOMC_CFG_SENSORS_LIGHT_AS3676_PATH),)
 $(SOMC_CFG_SENSORS_LIGHT_AS3676)-cflags += -DALS_PATH=\"$(SOMC_CFG_SENSORS_LIGHT_AS3676_PATH)\"
-- 
1.7.10.4

